schemas/fingerprint-related-visitors-api.yaml

```diff
@@ -0,0 +1,93 @@
+openapi: 3.0.3
+info:
+  title: Related Visitors API
+  summary: |
+    Use this API to unify and personalize the experience for your anonymous users who visit your website from regular and in-app browsers. 
+  version: '1.0'
+  contact:
+    name: Fingerprint Support
+    email: support@fingerprint.com
+tags:
+  - name: Related Visitors
+servers:
+  - url: https://api.fpjs.io
+    description: Recommended if your customer base is primarily in the Americas.
+  - url: https://eu.api.fpjs.io
+    description: Recommended if your customer base is primarily in Europe.
+  - url: https://ap.api.fpjs.io
+    description: Recommended if your customer base is primarily in the APAC region.
+securityDefinitions:
+  APIKeyHeader:
+    type: apiKey
+    in: header
+    name: Auth-API-Key
+  APIKeyQueryParam:
+    type: apiKey
+    in: query
+    name: api_key
+security:
+  - APIKeyHeader: []
+  - APIKeyQueryParam: []
+paths:
+  /related-visitors?visitor_id={visitor_id}:
+    get:
+      tags:
+        - Related Visitors
+      operationId: 'getRelatedVisitors'
+      summary: Get Related Visitors
+      description: |
+        This API will enable you to find if one or more web and in-app browser sessions originated from the same mobile device.
+        When requested, this API will search identification events within the past 6 months to find the visitor IDs that belong to the same mobile device as the given visitor ID.
+        Please visit the [Overview](https://dev.fingerprint.com/reference/related-visitors-api) page to learn more about this API.
+      parameters:
+        - name: visitor_id
+          description: The [visitor ID](https://dev.fingerprint.com/docs/js-agent#visitorid) for which you want to find the other visitor IDs that originated from the same mobile device.
+          in: query
+          required: true
+          schema:
+            type: string
+      responses:
+        '200':
+          description: OK.
```

Comment:
This API returns empty response?

Reply:
No :-) Copy and Paste error.
I have tried to provide a proper schema now.